src/app/backend/db/util.ts

```diff
@@ -14,9 +14,10 @@ INSERT INTO fhir_servers (
   scopes,
   access_token,
   token_expiry,
+  ca_cert,
```

Comment:
do we want to base64 encode these values when we're storing them? ran into issues with the newlines in CI when I was dealing with the variables, so wondering if it'll be simpler to store them as contiguous strings and then unencode them in the interface body when we send off the request

Reply:
Since they're just being stored in the database, I don't think we'll run into the same issue, right? For tests, we can just use template strings hardcoded in the test and I think it should work okay. But I guess I should get the tests passing properly before I decide if it works 😅